CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ and this project adheres to
 
 ## [Unreleased]
 
+### Added
+
+- Updated SummaryList so that a row's value can be a ReactNode rather than just
+  a string
+- Updated SummaryList to allow actions to appear in summary lists
```

Comment:
I'm debating whether either of these belongs under a `Changed` heading... I guess they're adding features to existing features, so `Added` is probably correct.